src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator: do work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator: stop work");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. Consider changing it to something like "Excavator stopped its work." to clearly indicate the action being taken.